apify-docs-theme/src/theme/custom.css

```diff
@@ -393,36 +592,36 @@ html.plugin-docs .theme-doc-markdown {
     margin-top: var(--ifm-list-item-margin);
 }
 
-html.plugin-docs .theme-doc-markdown img {
+html .plugin-docs .theme-doc-markdown img {
     display: block;
-    margin: 1em auto 2em;
+    margin: 16px auto 32px;
 }
 
-html.plugin-docs .theme-doc-markdown h1 {
+html .plugin-docs .theme-doc-markdown h1 {
     font-weight: 600;
     font-size: 48px;
     line-height: 64px;
     color: #000;
 }
 
-html[data-theme='dark'].plugin-docs .theme-doc-markdown h1 {
+html[data-theme='dark'] .plugin-docs .theme-doc-markdown h1 {
```

Comment:
This looks wrong, the plugin-docs class is in the html element unless you changed something.

Reply:
I thought so too but then I noticed it without the spaced it overrides all H1s everywhere and with the space it affects only the `.plugin-docs class`